src/stream/src/executor/managed_state/top_n/top_n_state_new.rs

```diff
@@ -70,13 +70,14 @@ impl<S: StateStore> ManagedTopNStateNew<S> {
                 ColumnDesc::unnamed(ColumnId::from(id as i32), data_type.clone())
             })
             .collect::<Vec<_>>();
-        let state_table = StateTable::new_without_distribution(
+        let mut state_table = RowBasedStateTable::new_without_distribution(
             store,
             table_id,
             column_descs,
             order_types,
             pk_indices,
         );
+        state_table.disable_sanity_check();
```

Comment:
Why do we need to disable sanity check for top-n?

Reply:
`disable_sanity_check()` uses point get of `StorageTable`, which is not compatible with row-based yet, let's remove it after the refactor of point-get.